docs/contributors/code/react-native/osx-setup-guide.md

```diff
@@ -36,7 +36,7 @@ If you have an existing Gutenberg checkout be sure to fully clean `node_modules`
 This may help avoid errors in the future.
 
 ```sh
-npm run distclean
+npm run other:distclean
```

Comment:
`distclean` is a common name used in projects for removing generated folders/files so it could stay as is.

Reply:
Fixed with e2dd094👍